_posts/shaders/2019-09-09-guide_parallax_corrected_cubemaps.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Parallax-Corrected Cubemaps
-permalink: /guide/parallax-corrected-cubemaps/
-category: guide
+permalink: /shader/parallax-corrected-cubemaps/
+category: shader
 tags:
   - hammer
```

Comment:
Best to replace this with `mapping` tag, as this is the only thing on the site with this tag and it's somewhat synonymous with hammer

Reply:
Resolved in [`36540aa` (#233)](https://github.com/momentum-mod/docs/pull/233/commits/36540aad6a67d20708e15095329ae57c3da3fe72)